app/src/org/commcare/dalvik/application/CommCareApplication.java

```diff
@@ -198,6 +210,22 @@ public void onCreate() {
 
         //The fallback in case the db isn't installed 
         resourceState = initializeAppResources();
+
+        ACRA.init(this);
+        this.initACRA();
+    }
+
+    public void initACRA(){

```

Comment:
This can be static and live in ACRAUtils. (Just have it take a Context argument)